server/indexers/schema.sql

```diff
@@ -3,6 +3,7 @@ CREATE TABLE "public"."auction" (
     "startTime" int4 NOT NULL,
     "endTime" int4 NOT NULL,
     "winner" text,
+    "owner" text,
```

Comment:
You are correct in your DM that `owner` belongs to the `noun` table, not `auction`